packages/@react-spectrum/steplist/intl/en-US.json

```diff
@@ -0,0 +1,6 @@
+{
+  "completed": "Completed: ",
+  "notCompleted": "Not completed: ",
+  "current": "Current: ",
+  "steplist": "Step List"
```

Comment:
don't think we need this line since the aria hook the the one that needs this translation?

Reply:
This is for localizing the VisuallyHidden component states of the step to the DOM:

```
<li>
  <a role="link" tabIndex="0">
    <VisuallyHidden>Completed: </VisuallyHidden>
    Details
  </a>
</li>
```

Reply:
Right, but we don't need `"steplist"` if I'm not mistaken? Only the previous three lines are used: https://github.com/adobe/react-spectrum/blob/060af1459b91d76a15d6d165f0d6b6902e222c4f/packages/%40react-spectrum/steplist/src/StepListItem.tsx#L55-L61